ext/drsyscall/drsyscall.dox

```diff
@@ -1,137 +0,0 @@
-/* **********************************************************
```

Comment:
Is it going to be confusing that every file was copied over except this one?  Better to leave it and either keep the gendox exclusion, or what I vote for is to keep it and have it be in the DR docs: you have the note that they are duplicated.  So it seems best to keep this.  Then removing from drmemory is easier and we don't have to remember there's one file that was lost.